diff --git a/setup.py b/setup.py
new file mode 100644
index 0000000..66589e0
--- /dev/null
+++ b/setup.py
@@ -0,0 +1,35 @@
+from setuptools import setup
+
+
+setup(
+    name="backoff",
+    version="1.11.1",
+    description="Function decoration for backoff and retry",
+    author="Bob Green",
+    author_email="<rgreen@aquent.com>",
+    url="https://github.com/litl/backoff",
+    license="MIT",
+    keywords=["retry", "backoff", "decorators"],
+    classifiers=[
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Developers",
+        "Programming Language :: Python",
+        "License :: OSI Approved :: MIT License",
+        "Natural Language :: English",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Topic :: Internet :: WWW/HTTP",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+        "Topic :: Utilities",
+    ],
+    packages=["backoff"],
+    python_requires=">=3.5",
+)
